src/coreclr/jit/emit.cpp

```diff
@@ -6700,7 +6700,7 @@ unsigned emitter::emitEndCodeGen(Compiler* comp,
         if (emitComp->opts.disAsm)
         {
             printf("\n%s:", emitLabelString(ig));
-            if (!emitComp->opts.disDiffable)
+            if (!emitComp->opts.disDiffable && emitComp->opts.disRawInstr)
```

Comment:
It does not really make sense to me that this goes under `disRawInstr`.

Reply:
specially it is not added in DEBUG few lines above.